static/js/popovers.js

```diff
@@ -744,7 +744,7 @@ exports.register_click_handlers = function () {
         } else {
             user = people.get_by_email(email);
         }
-        show_user_info_popover(this, user, message);
+        show_sender_info_popover(this, user, message);
```

Comment:
I don't think this rename is accurate; the code path here is clicking on a mention of the user.  I'd prefer to just skip this commit.

Reply:
I had rename this because for bot owners case we have no related message. 
I agree that renaming to 'show_sender_info_popover` is not correct as it is also used for mentions.
But I think we still need to rename it to keep it separate from bot owners and invites case(which will be added soon)  and I am thinking of other names, do you have any suggestions for for the name?

Reply:
Till now, I have thought of `show_user_info_popover_for_message`, but I do think we can think of something better. Any suggestions from your side?